helix-query.yaml

```
@@ -21,6 +21,12 @@ indices:
       image:
         select: head > meta[property="og:image"]
         value: attribute(el, "content")
+      author:
+        select: head > meta[name="author"]
+        value: attribute(el, "content")
+      tags:
+        select: head > meta[property="article:tag"]
+        value: attribute(el, "content")
```

Do you want to split the tags in multiple values?

Would look something like this:
```suggestion
        values: |
          match(attribute(el, 'content'), '([^,]+)')
```